### DISCUSSION TITLE
How to specify information record in result end (MongoDB)?

### DISCUSSION THREAD

**Discussion by w4230213** [celery/celery#8059] [General]
My project combined FastAPI + Celery, rabbitmq as broker and mongoDB as result end, with 4 distributed worker hosts do the real task job, and a single Celery instance within Web server to send jobs.
Now I wanna know which worker actually consumed the task, since the results will all be store in Mongo,  is there a way to specify the meta information record in MongoDB? So I can get an unified entry point to do the search work in one place.
The default keys are ['_id', 'status', 'result', 'traceback', 'children', 'date_done']. Could it add some more key like "Worker" / "Hostname"?

**Comment by thedrow** [comment]
The `result_extended` setting allows saving more information such as the [hostname](https://github.com/celery/celery/blob/main/celery/backends/base.py#L488) in the result backend.
You should set it to `True`.
